ale_linters/erlang/dialyzer.vim

```diff
@@ -0,0 +1,74 @@
+" Author: Autoine Gagne - https://github.com/AntoineGagne
+" Description: Define a checker that runs dialyzer on Erlang files.
+
+function! ale_linters#erlang#dialyzer#FindPlt() abort
+    let l:plt_file = split(globpath('_build', '**/*_plt'), '\n')
```

Comment:
Do Erlang projects always or often have `_build` directories? You should use a function like `ale#path#FindNearestDirectory` to find a directory based on where the buffer is. The use of `globpath` with `**` could cause major performance issues. I've seen bug reports where using `globpath` to search through directories in large directories has slowed Vim significantly.

Reply:
The `_build` directory is not always present. It is present when the `rebar3` build tool is used (it is the most used build system in the ecosystem). 

I used the wildcard `**` because I didn't know which profile was used when generating the `*_plt` file but I changed the code so a profile can be configured thus discarding the need of having to use the wildcard to recursively search.